src/coreclr/src/debug/dbgutil/elfreader.cpp

```diff
@@ -483,6 +539,9 @@ ElfReader::EnumerateProgramHeaders(Elf_Phdr* phdrAddr, int phnum, uint64_t baseA
         *ploadbias = loadbias;
     }
 
+    m_elfLoadMin = baseAddress;
```

Comment:
EnumerateProgramHeaders isn't so suppose to cache any state in the class because of the way it is used in createdump. Only the symbol lookup functions on top of it can cache data.

Reply:
That is why EnumerateProgramHeaders has out parameters (ploadbias/pdynamicAddr, etc).